src/App.js

```diff
@@ -9,7 +11,15 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <main className="main">
+    <h1 className="title">React Article</h1>
+    <Article
+      title={article.title}
+      text={article.text}
+      date={article.date}
+    />
```

Comment:
```suggestion
    <Article {...article} />
```

it's correct, but also you can use spread operator for this😉
https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493